provisioning/miniprov/hare_mp/main.py

```diff
@@ -601,6 +601,18 @@ def generate_support_bundle(args):
         if args.t:
             cmd.append('-t')
             cmd.append(args.t[0])
+        if args.duration:
+            print("Duration: ", args.duration)
+        if args.size_limit:
+            print("Size: ", args.size_limit)
```

Comment:
For size limit, you can implement a simple function to truncate log files from the start, like a tail command. I think this can be done in a separate task.

Reply:
Yes, we're going to work on another task. You can check on this https://jts.seagate.com/browse/EOS-25684
Thanks!